lib/ucsc/metadata_schema.rb

```diff
@@ -2,17 +2,17 @@ module Ucsc
   class MetadataSchema < ScoobySnacks::MetadataSchema
 
     def self.custom_display_groups
-      [:admin_only]
+      [:admin_only, :add_parent_value]
```

Comment:
Comment here on the choice to add this to display groups, with a "TODO" note for updating scoobysnacks with ADD as a boolean when time allows.

Reply:
Done. Thank you.